src/analyzer.js

```diff
@@ -169,7 +168,7 @@ function readStatsFromFile(filename) {
 }
 
 function getChildAssetBundles(bundleStats, assetName) {
-  return _.find(bundleStats.children, (c) =>
+  return bundleStats.children.find((c) =>
```

Comment:
Are you sure `bundleStats.children` is always defined?

Reply:
Nope, relied on tests. Will coalesce with []